framework/contracts/native/ibc-host/tests/integration.rs

```diff
@@ -175,7 +181,7 @@ fn account_creation_full() -> anyhow::Result<()> {
         ConfigResponse {
             ans_host_address: abstr_origin.ans_host.address()?,
             version_control_address: abstr_origin.version_control.address()?,
-            account_factory_address: abstr_origin.account_factory.address()?,
+            module_factory_address: abstr_origin.module_factory.address()?,
```

Comment:
Why is module_factory now included in the config response ?

Reply:
Account need it during instantiation, it was forwarded by account factory before